apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdAll.json

```diff
@@ -2730,6 +2730,10 @@
         },
         "productOrService" : {
           "coding" : [ {
+            "system" : "https://bluebutton.cms.gov/resources/variables/hcpcs_cd",
+            "code" : "M55"
+          },
+          {
             "system" : "https://bluebutton.cms.gov/resources/codesystem/hcpcs",
             "code" : "M55"
           } ]
```

Comment:
Would this not still trigger the filtering?  The samhsa logic seems to iterate over each coding in the `productOrService`. https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4SamhsaMatcher.java#L528-L543

Reply:
more changes to come ; converted to draft for now